src/util/validateJson.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint @typescript-eslint/no-explicit-any: "off" */
+/* eslint @typescript-eslint/explicit-module-boundary-types: "off" */
+import type {
+  FeatureCollection,
+  Feature,
+  GeoJsonProperties,
+  Geometry
+} from 'geojson';
+
+export function isFeatureCollection(object: any): object is FeatureCollection {
+  return (
+    typeof object !== 'undefined' &&
+    typeof object === 'object' &&
+    object.type === 'FeatureCollection' &&
+    Array.isArray(object.features) &&
+    object.features.every(isFeature)
+  );
+}
+
+export function isFeature(object: any): object is Feature {
+  return (
+    typeof object !== 'undefined' &&
+    typeof object === 'object' &&
+    object.type === 'Feature' &&
+    (typeof object.id === 'string' ||
+      typeof object.id == 'number' ||
```

Comment:
```suggestion
      typeof object.id === 'number' ||
```

Reply:
Fixed this! ...by removing it entirely 😏